controllers/common.go

```diff
@@ -127,13 +127,13 @@ func isAivenServerError(err error) bool {
 }
 
 // NewAivenClient returns Aiven client (aiven/aiven-go-client/v2)
-func NewAivenClient(token string) (*aiven.Client, error) {
-	return aiven.NewTokenClient(token, "k8s-operator/"+version)
+func NewAivenClient(token, kubeVersion, operatorVersion string) (*aiven.Client, error) {
+	return aiven.NewTokenClient(token, fmt.Sprintf("k8s-operator/%s/%s", kubeVersion, operatorVersion))
 }
 
 // NewAivenGeneratedClient returns Aiven generated client client (aiven/go-client-codegen)
-func NewAivenGeneratedClient(token string) (avngen.Client, error) {
-	return avngen.NewClient(avngen.TokenOpt(token), avngen.UserAgentOpt("k8s-operator/"+version))
+func NewAivenGeneratedClient(token, kubeVersion string) (avngen.Client, error) {
```

Comment:
The operatorVersion seems to be missing here, error in basic_controller.go:

![image](https://github.com/aiven/aiven-operator/assets/25483483/62d14ae3-2745-4938-bc3d-3ee321e4f775)